NYTimes.com.js

```diff
@@ -340,6 +249,102 @@ var testCases = [
 				"seeAlso": []
 			}
 		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.nytimes.com/2015/05/10/nyregion/manicurists-in-new-york-area-are-underpaid-and-unprotected.html?_r=0",
+		"items": [
+			{
+				"itemType": "newspaperArticle",
+				"title": "반짝이는 매니큐어에 숨겨진 네일 미용사들의 어두운 삶",
+				"creators": [
+					{
+						"firstName": "Sarah Maslin",
+						"lastName": "Nir",
+						"creatorType": "author"
+					}
+				],
+				"date": "2015-05-07",
+				"ISSN": "0362-4331",
+				"abstractNote": "뉴욕타임스는 취재 중 많은 네일숍 직원들이 부당한 대우와 인종차별 및 학대에 흔하게 시달리며 정부 노동자법률기구의 보호도 제대로 받지 못한다는 사실을 발견했다.",
+				"archiveLocation": "100000003670004",
+				"language": "ko-KR",
+				"libraryCatalog": "www.nytimes.com",
+				"publicationTitle": "The New York Times",
+				"section": "N.Y. / Region",
+				"url": "https://www.nytimes.com/2015/05/10/nyregion/manicurists-in-new-york-area-are-underpaid-and-unprotected.html",
+				"attachments": [
+					{
+						"title": "Snapshot"
+					}
+				],
+				"tags": [
+					"Beauty Salons",
+					"Discrimination",
+					"Korean-Americans",
+					"Labor and Jobs",
+					"New York City",
+					"Wages and Salaries"
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.nytimes.com/2017/05/24/us/politics/russia-trump-manafort-flynn.html?hp&action=click&pgtype=Homepage&clickSource=story-heading&module=span-ab-top-region&region=top-news&WT.nav=top-news&_r=0",
+		"items": [
+			{
+				"itemType": "newspaperArticle",
+				"title": "Top Russian Officials Discussed How to Influence Trump Aides Last Summer",
+				"creators": [
+					{
+						"firstName": "Matthew",
+						"lastName": "Rosenberg",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Adam",
+						"lastName": "Goldman",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Matt",
+						"lastName": "Apuzzo",
+						"creatorType": "author"
+					}
+				],
+				"date": "2017-05-24",
+				"ISSN": "0362-4331",
+				"abstractNote": "American spies collected intelligence last summer revealing that Russians were debating how to work with Trump advisers, current and former officials say.",
+				"archiveLocation": "100000005126615",
+				"language": "en-US",
+				"libraryCatalog": "www.nytimes.com",
+				"publicationTitle": "The New York Times",
+				"section": "Politics",
+				"url": "https://www.nytimes.com/2017/05/24/us/politics/russia-trump-manafort-flynn.html",
+				"attachments": [
+					{
+						"title": "Snapshot"
+					}
+				],
+				"tags": [
+					"Cyberwarfare and Defense",
+					"Donald J",
```

Comment:
this makes sense to me -- might also be worthwhile to look at `name="article:tag"` and prefer over `name="keywords"` in EM